packages/atomic/src/components/search/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -148,16 +148,20 @@ export class AtomicSegmentedFacet
     return <b class="inline-block my-3 mx-2">{this.label}:</b>;
   }
 
-  public render() {
-    if (
-      this.searchStatus.state.hasError ||
-      !this.facetState.values.length ||
-      !this.facet.state.enabled
-    ) {
-      return <Hidden></Hidden>;
-    }
-
+  renderLoadingAnimation() {
     return (
+      <div
+        part="placeholder"
+        aria-hidden
+        class="h-8 my-2 w-screen bg-neutral animate-pulse"
```

Comment:
would be nice if the width is related to the number of values of the facet

Reply:
I can do that !

Reply:
This is a bit more complex than what I thought. I will create a Jira for it.